core/src/main/java/org/apache/carbondata/core/memory/CarbonUnsafe.java

```diff
@@ -34,6 +35,9 @@
 
   public static final int DOUBLE_ARRAY_OFFSET;
 
+  public static final boolean LITTLEENDIAN =
```

Comment:
It is `static final` so better to keep in caps,  I will rename to `ISLITTLEENDIAN`